### ISSUE TITLE
Gradient as backgroundColor

### ISSUE THREAD

**Issue by asmiralic** [letsar/flutter_slidable#428]
Will it ever be implemented possibility to add Linear Gradient as background in SlidableAction()??

**Comment by davidBarbieri** [comment]
I agree on this issue, in the old version the buttons is the panel where fully customizable. I had to remove all the styles in the panel, also i was using widget as SlidableAction that contained thre reusable graphics and logic for the click event.
Would be nice to have some room for customization
